External/FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -343,6 +343,9 @@ namespace FEXCore::Context {
       // Walk the threads and tell them to clear their caches
       // Useful when our block size is set to a large number and we need to step a single instruction
       for (auto &Thread : Threads) {
+        // Wait for thread to be fully constructed
+        // XXX: Look into thread partial construction issues
```

Comment:
Are you planning on fixing this in this PR? If not, do you mind elaborating this comment for future reference?

Reply:
This is for future reference. It's an unrelated, pre-existing bug/gotcha that I run into while working on other things.

Reply:
(And I just went in all uses I found that didn't check for the condition and added these comments so at least we can spot them)

Reply:
@neobrain I've made follow up #1696 for an in general better handling of these todos